File: 2.0/java/keywords/findKeywordLeases.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.common.model.Page;
import com.callfire.api.client.api.common.model.request.CommonFindRequest;
import com.callfire.api.client.api.keywords.model.KeywordLease;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        CommonFindRequest request = CommonFindRequest.create()
            .offset(0L)
            .limit(10L)
            .fields("items(keyword,leaseBegin,leaseEnd)")
            .build();
        Page<KeywordLease> leases = client.keywordLeasesApi().find(request);
    }
}

File: 2.0/csharp/keywords/findKeywordLeases.cs
using CallfireApiClient;
using CallfireApiClient.Api.Common.Model;
using CallfireApiClient.Api.Common.Model.Request;
using CallfireApiClient.Api.Keywords.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new CommonFindRequest
        {
            Offset = 0,
            Limit = 10,
            Fields = "items(keyword,leaseBegin,leaseEnd)"
        };
        Page<KeywordLease> leases = client.KeywordLeasesApi.Find(request);
    }
}
